.travis.yml

```diff
@@ -1,19 +1,22 @@
 sudo: required
-
 language: go
-
 go:
-  - 1.8
-
+- 1.9
 services:
-  - docker
-
+- docker
 before_install:
-  - sudo apt-get install jq
+- sudo apt-get install jq
 
-script:
-  - make unittest
-  - ./scripts/ci-smoketest.sh
+stage: Build Release
+script: make github-release
 
-after_script:
-  - make destroy-smoketest
+deploy:
```

Comment:
We should have the team chime in, but I don't think we want/need to release the binaries on Github. We already have an easier pathway for users to download from S3 (because they can get the release for their specific OS), and adding that here just seems redundant and more code to maintain.

Reply:
Another note, we don't want to run this in Travis. For one thing, Travis doesn't have the permissions to push to S3 or our Dockerhub account. Also, if we did go the Travis route for releases, we should run system tests first, which is a very large unit of work. Any Travis related stuff should be dropped from this PR.

Reply:
Oooh, great point about travis' permissions - I hadn't even thought about that.

As for releasing binaries on GitHub in addition to our docs, I'm sort of on the fence. I think that if we _do_ release on GitHub as well, we should definitely break it up by OS in the same way we do in the docs. But whether or not we _should_ do that... I don't know just yet. I like the idea of having binary releases so strongly coupled to the tagged version notes; I think that it's a good pattern found in a lot of other OSS repositories. It is redundant, though, given that we also link to binaries stored in S3 in our docs. Maybe the release process could be that the version notes on GitHub and the download links in our docs both link to the same source of binaries?

I guess what I like about releasing binaries on GitHub is that we're not tied to AWS resources paid for by IQVIA - if anything goes south and we lose access to those resources, our OSS tool is unaffected and still available on GitHub.

Reply:
Great thoughts everyone, just to clarify with the deploy portion. 
It'd only be building the binaries the current layer0.ims.io/#download which can be done on in travis. 

As far as s3 and DockerHub go we can continue to do that locally or figure out if it can be deployed via travis, which I believe can be done, ie https://docs.travis-ci.com/user/deployment/s3/

Reply:
I guess my concern is more about releasing in multiple locations. To tlake's point, releasing on Github makes a lot of sense, and if we went that route, I'd like to see the S3 stuff go away.